gpio-adapter.js

```diff
@@ -173,7 +174,17 @@ class GpioAdapter extends Adapter {
     }
 
     if (manifest.moziot.config.hasOwnProperty('gpios')) {
-      gpios = Object.assign(gpios, manifest.moziot.config.gpios);
+      if (Array.isArray(manifest.moziot.config.gpios)) {
+        for (const gpio of manifest.moziot.config.gpios) {
+          gpios[gpio.pin.toFixed(0).toString()] = {
+            name: gpio.name,
+            direction: gpio.directioin,
+            value: gpio.value,
+          };
+        }
+      } else {
+        gpios = Object.assign(gpios, manifest.moziot.config.gpios);
```

Comment:
It would be useful to add a comment to indicate that the else path is for backwards compatability, and that the if path is the new "normal".

Reply:
Done.